src/pyrobot/habitat/base.py

```diff
@@ -52,8 +54,8 @@ def get_state(self, state_type="odom"):
         (r, pitch, yaw) = euler_from_matrix(cur_rotation, axes="sxzy")
         # Habitat has y perpendicular to map where as ROS has z perpendicular
         # to the map. Where as x is same.
-        # Here ROS_Z = -1 * habitat_z and ROS_Y = habitat_x
-        return (-1 * true_position[2], true_position[0], yaw)
+        # Here ROS_X = -1 * habitat_z and ROS_Y = -1*habitat_x
+        return (-1 * true_position[2], -1* true_position[0], yaw)
 
     def stop(self):
         raise NotImplementedError("Veclocity control is not supported in Habitat-Sim!!")
```

Comment:
Nope, it is supported and now documented 😄 -- https://aihabitat.org/docs/habitat-sim/rigid-object-tutorial.html#velocitycontrol

Reply:
thanks, that's probably next to fill out, at some point :)